newt/settings.py

```diff
@@ -215,8 +215,7 @@
 NEWT_HOST = 'localhost'
 NEWT_DOMAIN = 'nersc.gov'
 NEWT_COOKIE_LIFETIME = 43200
-MYPROXY_SERVER = 'nerscca2.nersc.gov'
-
+SSHPROXY_SERVER = 'https://sshproxy.nersc.gov/create_pair/default/'
```

Comment:
I think we want to have both MYPROXY_SERVER and SSHPROXY_SERVER settings available, depending on the chosen backend. Could you include both?